recipes/probablepeople/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "probablepeople" %}
+{% set version = "0.5.2" %}
+{% set sha256 = "c3d7e41dd7bd0110553efa972d0e043b712e6bf9f5c20f9b7064f1e5003c6864" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - python-crfsuite >=0.8
+    - probableparsing
+    - future
+    - doublemetaphone
```

Comment:
These are specified in `setup.py`. I'm now verifying whether they are needed or not.